db/schema.rb

```diff
@@ -308,4 +308,34 @@
       rules.identifier
      FROM rules;
   SQL
+  create_view "tailorings", sql_definition: <<-SQL
+      SELECT profiles.id,
+      profiles.policy_id,
+      profiles.parent_profile_id AS profile_id,
+      profiles.value_overrides,
+      profiles.created_at,
+      profiles.updated_at
+     FROM profiles
+    WHERE (profiles.parent_profile_id IS NOT NULL);
+  SQL
+  create_view "v2_policies", sql_definition: <<-SQL
+      SELECT policies.id,
+      policies.name AS title,
+      policies.description,
+      policies.compliance_threshold,
+      business_objectives.title AS business_objective,
+      tailorings.profile_id,
+      COALESCE(policy_hosts.host_count, (0)::bigint) AS host_count,
+      policies.account_id
+     FROM (((policies
+       LEFT JOIN business_objectives ON ((business_objectives.id = policies.business_objective_id)))
+       JOIN ( SELECT DISTINCT ON (tailorings_1.policy_id) tailorings_1.profile_id,
+              tailorings_1.policy_id
+             FROM tailorings tailorings_1
+            ORDER BY tailorings_1.policy_id) tailorings ON ((tailorings.policy_id = policies.id)))
```

Comment:
This can be simplified by https://github.com/RedHatInsights/compliance-backend/pull/1852